base/task.jl

```diff
@@ -365,6 +365,60 @@ function wait(t::Task)
     nothing
 end
 
+# Wait multiple tasks
+waitany(tasks) = _wait_multiple(tasks)
+waitall(tasks; failfast=false) = _wait_multiple(tasks; all=true, failfast=failfast)
+
+function _wait_multiple(waiting_tasks::Union{AbstractVector{Task},AbstractSet{Task},Tuple{Task}}; all=false, failfast=false)::NTuple{2,Vector{Task}}
```

Comment:
Also, I wonder if the return type declaration is redundant, given that this function seems type stable with respect to the return values.

Reply:
Yes, its my fault. I'll remove the return type.